### PR TITLE
Feature update for lts9

### LINKED ISSUE #28: Update to be compatible with SonarQube LTS 9

### PR DESCRIPTION
**Pull request by diegorodriguez31**
## Proposed changes

Update all tools with their latest version to be compatible with SonarQube LTS 9

## Types of changes

What types of changes does your code introduce to this project?

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

## Issues closed by changes

- [x] Close #28 

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I agree with the [CODE OF CONDUCT](CODE_OF_CONDUCT.md)
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: tests/test_cnes_sonar_scanner.py [hunk 1]
```diff
@@ -52,8 +52,70 @@ class TestCNESSonarScanner:
...
+                    })
+        if cls.RUN:
+            print(f"Stopping {cls.SONARQUBE_CONTAINER_NAME}...")
+            docker_client = docker.from_env()
+            docker_client.containers.get(cls.SONARQUBE_CONTAINER_NAME).stop()
+            print(f"Removing bridge network {cls.SONARQUBE_NETWORK}...")
+            docker_client.networks.get(cls.SONARQUBE_NETWORK).remove()
 
     # Functions
+    def get_sonarqube_token(cls):
```
Comment (louisjdmartin) [inline tests/test_cnes_sonar_scanner.py:110]: Si "self" n'est pas utile alors il faut ajouter un `@classmethod`

File: tests/test_cnes_sonar_scanner.py [hunk 2]
```diff
@@ -52,8 +52,70 @@ class TestCNESSonarScanner:
     SONARQUBE_NETWORK = os.environ.get("SONARQUBE_NETWORK", "sonarbridge")
     _SONAR_SCANNER_IMAGE = "lequal/sonar-scanner"
     _PROJECT_ROOT_DIR = str(Path(os.getcwd()).parent)
+    SONARQUBE_TOKEN = ""
```
Comment (louisjdmartin) [inline tests/test_cnes_sonar_scanner.py:55]: Ok pour ignorer cette erreur Codacy

**Review by louisjdmartin — CHANGES_REQUESTED** [review]
Ok pour ignorer les erreurs type "info" de codacy.

Eventuellement, si il est possible de monter (facilement) la limite de caractères par ligne a 130 je te laisse le faire

**Comment by diegorodriguez31** [comment]
> Ok pour ignorer les erreurs type "info" de codacy.
> 
> Eventuellement, si il est possible de monter (facilement) la limite de caractères par ligne a 130 je te laisse le faire

Malheureusement, ce n'est pas possible de facilement remonter la limite du nombre de ligne sur Codacy.

**Review by louisjdmartin — APPROVED** [review]

**Pull request merged**